### PR TITLE
[quest] Added 25241 The Land, Corrupted

### PR DESCRIPTION
**Pull request by TorpedoCXC**
Also fixed the alliance version (25397) of this quest.

## Screenshots
WoWScrnShot_100424_134710

### CODE REVIEW

File: Database/Corrections/cataQuestFixes.lua [hunk 1]
```diff
@@ -3362,6 +3362,10 @@ function CataQuestFixes.Load()
             [questKeys.preQuestSingle] = {},
             [questKeys.objectives] = {{{39464,nil,Questie.ICON_TYPE_EVENT}}},
         },
+        [25241] = { -- The Land, Corrupted
+            [questKeys.requiredRaces] = raceKeys.ALL_HORDE,
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:3366]:
```suggestion
```

File: Database/Corrections/cataQuestFixes.lua [hunk 2]
```diff
@@ -3561,6 +3565,7 @@ function CataQuestFixes.Load()
             [questKeys.preQuestSingle] = {25394},
         },
         [25397] = { -- The Land, Corrupted
+            [questKeys.requiredRaces] = raceKeys.ALL_ALLIANCE,
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:3568]:
```suggestion
```

**Review by BreakBB — COMMENTED** [review]
Same as with the other PR, both quests are already correctly marked as Horde/Alliance quest 🤔

**Comment by TorpedoCXC** [comment]
Same as here: https://github.com/Questie/Questie/pull/6462#issuecomment-2395684278

Sorry and thx!

**Pull request merged**